freon@UshakovVasilii_Github.yahoo.com/prefs.js

```diff
@@ -26,82 +26,161 @@ var FreonPrefsWidget = new GObject.registerClass(class Freon_FreonPrefsWidget ex
-            key: 'unit', y : i++, x : 3,
-            label: _('Temperature Unit')
+            label: _('Position in Panel'),
+            items : {left : _('Left'), center : _('Center'), right : _('Right')},
+            key: 'position-in-panel', y : i++, x : j
         });
 
-        this._addComboBox({
-            items : {left : _('Left'), center : _('Center'), right : _('Right')},
-            key: 'position-in-panel', y : i, x : 0,
-            label: _('Position in Panel')
+        this._addLabel({
+            label: _('Index in Panel'),
+            y : i, x : j
         });
 
         let panelBoxIndex = Gtk.SpinButton.new_with_range (-1, 20, 1);
-        this.attach(panelBoxIndex, 2, i, 1, 1);
+        this.attach(panelBoxIndex, j + 1, i++, 1, 1);
         this._settings.bind('panel-box-index', panelBoxIndex, 'value', Gio.SettingsBindFlags.DEFAULT);
 
-        this._addSwitch({key : 'show-degrees-on-panel', y : i++, x : 3,
-            label : _('Show \u00b0C/\u00b0F on Panel')});
+        this._addSwitch({key : 'show-icon-on-panel', y : i++, x : j,
+            label : _('Show Icon')});
 
-        this._addSwitch({key : 'show-icon-on-panel', y : i++, x : 3,
-            label : _('Show Icon on Panel')});
+        this._addComboBox({
+            label: 'Temperature Unit',
+            items : {centigrade : "\u00b0C", fahrenheit : "\u00b0F"},
+            key: 'unit', y : i++, x : j
+        });
 
-        this._addSwitch({key : 'show-fan-rpm', y : i, x : 0,
-            label : _('Show Fan Speed')});
+        this._addSwitch({key : 'show-degrees-on-panel', y : i++, x : j,
+            label : _('Show Temperature Unit')});
 
-        this._addSwitch({key : 'show-voltage', y : i++, x : 3,
-            label : _('Show Power Supply Voltage')});
+        this._addSwitch({key : 'show-rpm-on-panel', y : i++, x : j,
+            label : _('Show Fan Speed Unit')});
 
-        this._addSwitch({key : 'group-temperature', y : i, x : 0,
-            label : _('Group Temperature Items'),
-            help : _("Works if you have more than three temperature sensors")});
+        this._addSwitch({key : 'show-volts-on-panel', y : i++, x : j,
+            label : _('Show Voltage Unit')});
 
-        this._addSwitch({key : 'group-voltage', y : i++, x : 3,
-            label : _('Group Voltage Items'),
-            help : _("Works if you have more than three voltage sensors")});
+        this._addSwitch({key : 'show-decimal-value', y : i++, x : j,
+            label : _('Show Decimal Value'),
+            help : _("Show one digit after decimal")});
 
-        this._addComboBox({
-            items : {none : _('None'), hddtemp : 'Hddtemp', udisks2 : 'UDisks2', smartctl : 'smartctl', nvmecli : 'nvme-cli'},
-            key: 'drive-utility', y : i, x : 0,
-            label: _('HDD/SSD Temperature Utility')
+        i = 0;
+        j = 3;
+
+        this._addLabel({
+            label: _('Generic sensors'),
```

Comment:
```suggestion
            label: _('Generic Sensor Providers'),
```

I propose we consistently use the term "provider" to identify these various sources of sensor data.  "Backend" would also work, but I think "provider" may be more suitable for regular users; plus "backend" is way too overloaded, and backends can have backends which in turn also have backends...

Reply:
Done.